packages/cli/src/commands/dbCommands/studio.js

```diff
@@ -1,32 +1,26 @@
 import path from 'path'
 import fs from 'fs'
 
-import terminalLink from 'terminal-link'
-
 import { runCommandTask, getPaths } from 'src/lib'
+import { schema, epilogue } from 'src/commands/dbCommands/options'
 import c from 'src/lib/colors'
 
 export const command = 'studio'
 export const description = 'Start Prisma Studio'
 
 export const builder = (yargs) => {
-  yargs.epilogue(
-    `Also see the ${terminalLink(
-      'Redwood CLI Reference',
-      'https://redwoodjs.com/reference/command-line-interface#studio'
-    )}`
-  )
+  yargs.option('schema', schema()).epilogue(epilogue())
 }
 
-export const handler = async () => {
+export const handler = async ({ schema }) => {
   // No schema, no studio.
-  if (!fs.existsSync(getPaths().api.dbSchema)) {
+  if (!fs.existsSync(schema())) {
     console.log(
       `${c.warning(
         '[warning]'
       )} cannot start Prisma Studio; schema missing (${c.info(
         // So we're not hard coding schema.prisma's relative location
-        path.relative(getPaths().base, getPaths().api.dbSchema)
+        path.relative(getPaths().base, schema())
```

Comment:
```suggestion
        path.relative(getPaths().base, schema)
```

This should be a string.

Reply:
I think we return the absolute path for `schema` from `getPaths()` but a user would likely only supply a relative path. Do we have a test case that handles both of those?